app/views/articles/show.html.erb

```diff
@@ -229,6 +229,7 @@
 <div data-testid="flag-user-modal-container" class="flag-user-modal-container hidden"></div>
 
 <div class="fullscreen-code js-fullscreen-code"></div>
+<%= javascript_packs_with_chunks_tag "followButtons", defer: true %>
```

Comment:
The `followButtons ` pack is included in this file. Is there a container element in this file that requires a `data-follow-button-container="true"` to coordinate with the `followButtons` pack file?

Reply:
We only need to observe parts of the UI where follow buttons might be _dynamically_ added, and in most places where we use the pack they're just static. So we have quite a few places where the pack is included but there's no data attribute for 'watching'.

I've added a comment to the pack's mutation observer code to try to explain how this is being used